packages/extension-commerce-bm-seo/README.md

```diff
@@ -0,0 +1,91 @@
+# SEO Extension
+
+A [PWA Kit](https://github.com/SalesforceCommerceCloud/pwa-kit) extension that adds SEO functionality to your application through the [Shopper SEO URL Mapping API](https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-seo?meta=getUrlMapping). This extension provides features like:
+
+- Integrate routes managed in Business Manager with the PWA Kit
+- Integration with SCAPI Shopper SEO URL Mapping API
+- Support for multiple resource types (products, categories)
+
+## Installation
+
+```sh
+npm install @salesforce/extension-commerce-bm-seo
+
+# Also: 
+# - install the peer dependencies listed in the package.json
+# - see the Peer Dependancies section below for any other steps (e.g. make sure your app uses CommerceApiProvider component)
+```
+
+## Peer Dependencies
+
+PWA-Kit Application Extensions are NPM packages at their most simplest form, and as such you can define
+what peer dependencies are required when using it. Because this application extension provides
+Chakra UI via a page and components, it requires that the some peer dependencies are installed.
+
+Depending on what features your application extensions provides it's recommended you include any third-party
+packages as peer dependencies so that your base application doesn't end up having multiple versions of a 
+given package. See package.json for the full list of peer dependencies.
+
+## Configuration
+
+The SEO extension is configured via the `mobify.app.extensions` property in your config files or `package.json`:
```

Comment:
A lot of devs in our beta program where confused about where they can define their configuration. They assumed that it could only be defined in the package.json even tho we mention it being able to be defined in the config file. 

Maybe we can do something like:


```
The SEO extension is configured via the `app.extensions` property in your config (e.g. app/config/default.json) file or the key 'mobify.app/extensions' in your  `package.json`:
```